spec/support/with_client_authentication.rb

```diff
@@ -19,6 +19,9 @@
   end
 
   let(:auth_headers) { { 'Authorization': "Bearer #{@access_token}" } }
```

Comment:
Can't you reuse the :Authorization symbol you defined under here?

Reply:
Good call. `:Authorization` causes all sorts of problems when you try to reference it (because it's capitalised but not a constant), so I've just created another variable which we use in both instead :+1: